Library/Homebrew/dev-cmd/generate-formula-api.rb

```diff
@@ -69,11 +77,16 @@ def generate_formula_api
           File.write("api/formula/#{name}.json", FORMULA_JSON_TEMPLATE)
           File.write("formula/#{name}.html", html_template_name)
         end
+
+        homebrew_core_tap_hash["formulae"][formula.name] =
+          formula.to_hash_with_variations(hash_method: :to_api_hash)
```

Comment:
```suggestion
          formula.to_hash_with_variations(hash_method: :to_api_internal_v3_hash)
```

or similar? may also be worth renaming the existing one, too

Reply:
I doubt we'll run into a situation where we have more than 2 hash types we're generating with this method and if we do run into that problem, we can change the names then.